add-on/src/lib/ipfs-client/reloaders/index.js

```diff
@@ -0,0 +1,29 @@
+const { InternalTabReloader } = require('./internalTabReloader')
+const { LocalGatewayReloader } = require('./localGatewayReloader')
+const { WebUiReloader } = require('./webUiReloader')
+
+/**
+ * Prepares extension by creating an instance and awaiting for init.
+ *
+ * @param {ReloaderBase} extensions
```

Comment:
Where is `ReloaderBase` defined? You should use jsdocs `@typedef` to define it. 

Same for Browser & Logger if they're not already defined (tip: You can use `{import('./some/path').Namespace['property']}` if the type is defined elsewhere)

Reply:
Fixed.